### PR TITLE
Update release notes for 1.12.0.0

### PR DESCRIPTION
**Pull request by joshuali925**
*Issue #, if available:*

*Description of changes:*
add new PRs to release notes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: release-notes/opendistro-for-elasticsearch-sql.release-notes-1.12.0.0.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 ## 2020-11-20 Version 1.12.0.0
 
 ### Features
+* Add count() support for PPL ([#894](https://github.com/opendistro-for-elasticsearch/sql/pull/894))
+* Add like, isnull, isnotnull function in PPL ([#893](https://github.com/opendistro-for-elasticsearch/sql/pull/893))
+* Revert Error Messages for Run/Explain ([#889](https://github.com/opendistro-for-elasticsearch/sql/pull/889))
+* Extend the PPL identifier defintion ([#888](https://github.com/opendistro-for-elasticsearch/sql/pull/888))
+* Add identifier and datatype documentation for PPL ([#873](https://github.com/opendistro-for-elasticsearch/sql/pull/873))
+* Add Error Message for Explain and Run ([#872](https://github.com/opendistro-for-elasticsearch/sql/pull/872))
+* Revert java version in jdbc release workflow ([#871](https://github.com/opendistro-for-elasticsearch/sql/pull/871))
```
Comment (dai-chen) [inline release-notes/opendistro-for-elasticsearch-sql.release-notes-1.12.0.0.md:10]: remove this?

File: release-notes/opendistro-for-elasticsearch-sql.release-notes-1.12.0.0.md [hunk 2]
```diff
@@ -1,6 +1,14 @@
 ## 2020-11-20 Version 1.12.0.0
 
 ### Features
+* Add count() support for PPL ([#894](https://github.com/opendistro-for-elasticsearch/sql/pull/894))
+* Add like, isnull, isnotnull function in PPL ([#893](https://github.com/opendistro-for-elasticsearch/sql/pull/893))
+* Revert Error Messages for Run/Explain ([#889](https://github.com/opendistro-for-elasticsearch/sql/pull/889))
+* Extend the PPL identifier defintion ([#888](https://github.com/opendistro-for-elasticsearch/sql/pull/888))
+* Add identifier and datatype documentation for PPL ([#873](https://github.com/opendistro-for-elasticsearch/sql/pull/873))
```
Comment (dai-chen) [inline release-notes/opendistro-for-elasticsearch-sql.release-notes-1.12.0.0.md:8]: np: should be in documentation section?

**Review by dai-chen — APPROVED** [review]

File: release-notes/opendistro-for-elasticsearch-sql.release-notes-1.12.0.0.md (hunk 1 above)
Comment (davidcui-amzn) [reply release-notes/opendistro-for-elasticsearch-sql.release-notes-1.12.0.0.md:10]: Should this be removed? Or just moved to bug fix section?

**Review by davidcui-amzn — APPROVED** [review]

File: release-notes/opendistro-for-elasticsearch-sql.release-notes-1.12.0.0.md (hunk 1 above)
Comment (joshuali925) [reply release-notes/opendistro-for-elasticsearch-sql.release-notes-1.12.0.0.md:10]: I think we'll need to include all merged PRs, but I'll move it to infra section, thanks!

File: release-notes/opendistro-for-elasticsearch-sql.release-notes-1.12.0.0.md (hunk 2 above)
Comment (joshuali925) [reply release-notes/opendistro-for-elasticsearch-sql.release-notes-1.12.0.0.md:8]: good point I'll move it, thanks

**Pull request merged**